TS06-Facets-And-NDVI-Plotting-In-R.Rmd

```diff
@@ -224,300 +455,123 @@ tail(harMetDaily.09.11$month)
-
-plot.NDVI.2011 <- ggplot(NDVI.2011, aes(Date, meanNDVI))+
-  geom_point(colour = "forestgreen", size = 4) +
-  ggtitle("Daily NDVI at Harvard Forest, 2011")+
-  theme(legend.position = "none",
-        plot.title = element_text(lineheight=.8, face="bold",size = 20),
-        text = element_text(size=20))
- 
-#display the plots together
-grid.arrange(plot.par.2011, plot.NDVI.2011) 
-```
+#convert to date time - add a day "01" to each date to support this in base R
+met_monthly_HARV$date <- as.Date(paste(met_monthly_HARV$date,"-01",sep=""))
 
-This is nice but a bit confusing as the time on our x-axis doesn't fully line
-up.  To fix this we can assign the same min and max to both x-axes so that
-they align.  We can also label the axes 
+#add year
+met_monthly_HARV$year <- year(met_monthly_HARV$date)
 
-``` {r plot-same-xaxis}
-plot2.par.2011 <- plot.par.2011 +
-  scale_x_date(labels = date_format("%b %d"),
-               breaks = "3 months", minor_breaks= "1 week",
-               limits=c(min(NDVI.2011$Date),max=max(NDVI.2011$Date)))+
-  ylab("Total PAR") + xlab ("")
- 
+#add month
+met_monthly_HARV$month <- factor(month(met_monthly_HARV$date))
 
-plot2.NDVI.2011 <- plot.NDVI.2011 +
-  scale_x_date(labels = date_format("%b %d"),
-               breaks = "3 months", minor_breaks= "1 week",
-               limits=c(min(NDVI.2011$Date),max=max(NDVI.2011$Date)))+
-  ylab("Total NDVI") + xlab ("Date")
 
-
-grid.arrange(plot2.par.2011, plot2.NDVI.2011) 
-```
-
-#Challenge: Plot Air Temperature and NDVI
-Create a complementary plot pairing with Air Temperateure and NDVI.  Choose
-colors and symobls that show the data well.  Finally, plot air temperature, PAR,
-and NDVI in a single window to ease comparisons.  
-
-``` {r challengeplot-same-xaxis, echo=FALSE}
-plot.airt.2011 <- ggplot(harMet.daily2011, aes(date, airt))+
-  geom_point(colour="darkblue", na.rm=TRUE) +
-  ggtitle("Average Air Temperature\n Harvard Forest 2011")+
-  scale_x_date(labels = date_format("%b %d"),
-               breaks = "3 months", minor_breaks= "1 week",
-               limits=c(min(NDVI.2011$Date),max=max(NDVI.2011$Date)))+
-  ylab("Celcius") + xlab ("")+
-  theme(legend.position = "none",
-        plot.title = element_text(lineheight=.8, face="bold",size = 20),
-        text = element_text(size=20))
-
-grid.arrange(plot.airt.2011, plot2.NDVI.2011) 
-
-grid.arrange(plot2.par.2011, plot.airt.2011, plot2.NDVI.2011) 
+#create plot
+#rerun original par.precip code to incorporate the levels. 
+long_term_temp <- ggplot(met_monthly_HARV,aes(month, airt)) +
+         geom_point(na.rm=TRUE) +    #removing the NA values
+        ggtitle("Air Temperature 2001-2015 \n NEON Harvard Forest Site") +
+         theme(plot.title = element_text(lineheight=.8, face="bold",size = 20)) +
+         theme(text = element_text(size=20)) +
+         xlab("Total Precipitation (mm)") + ylab("Mean Total PAR")
+           
+#new facetted plots
+long_term_temp + facet_wrap(~ year, nc=3)
 ```
 
+***
 
```

Comment:
Cool! I like it too. Give it a run through if you have time and see if there are any kinks.
I'm pretty excited to teach these lessons!